hyperopt/tests/test_atpe_basic.py

```diff
@@ -1,19 +1,17 @@
 from __future__ import absolute_import
-import unittest
+from hyperopt import hp, fmin, atpe, space_eval
 
 
-class TestATPE(unittest.TestCase):
+class TestATPE:
     def test_run_basic_search(self):
         def objective(args):
             case, val = args
-            if case == "case 1":
+            if case == "case 2":
```

Comment:
why do you need this change? fwiw we don't need to have a realistic objective function here, just check that the end2end works

Reply:
Alright, so just `assert best` ?

Reply:
see comment above